src/sql/parts/query/editor/messagePanel.ts

```diff
@@ -62,6 +62,13 @@ const TemplateIds = {
 export class MessagePanelState {
 	public scrollPosition: number;
 	public collapsed = false;
+
+	constructor(@IConfigurationService configurationService: IConfigurationService) {
+		let collapsedSetting = configurationService.getValue<boolean>('sql.messagesDefaultOpen');
+		if (!isUndefinedOrNull(collapsedSetting)) {
+			this.collapsed = collapsedSetting;
```

Comment:
I think this logic is reversed. Won't this set `this.collapsed` to `true` when `sql.messagesDefaultOpen` is true? But we want it expanded in that case not collapsed